src/Middleware/ResponseCaching/src/ResponseCachingOptions.cs

```diff
@@ -1,10 +1,13 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.ComponentModel;
 
 namespace Microsoft.AspNetCore.ResponseCaching
 {
+    /// <summary>
+    /// Options for configuring the <see cref="ResponseCachingMiddleware"/>.
+    /// </summary>
     public class ResponseCachingOptions
```

Comment:
* For the `SizeLimit` property below, do you think it's valuable to explain what happens once the limit is hit?
* For `MaximumBodySize` should we add what happens if the limit is exceeded?

Reply:
Updated